### PR TITLE
[RA2 Ch4] updates to include NSX Container plugin(NCP)

### PR DESCRIPTION
**Pull request by xgao3**
The NSX Container Plugin (NCP) is a container plugin that integrates with NSX-T Data Center to build Kubernetes networking and security objects. The NCP can also serve as a metaplugin/CNI multiplexer that is capable of attaching multiple interfaces to Kubernetes Pods to provide multiple NIC container support

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -99,28 +99,30 @@ These specifications cover the [full lifecycle of a container](https://github.co
...
-| `req.inf.ntw.14` | Supported via an other CNI plugin | Supported via an other CNI plugin |
-| `req.inf.ntw.15` | Not relevant | Not relevant |
-| `req.inf.ntw.16` | Supported | Supported |
-| Cluster wide IP address management | Not suported | Supported |
-| Service based dicovery of all provisioned interfaces | Not supported | Supported |
+The used CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm) as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
+
+The NSX Container Plugin ([NCP](https://kubernetes.io/docs/concepts/cluster-administration/addons0/)) is a CNCF-compliant container plugin that integrates with NSX-T Data Center to build Kubernetes networking and security objects.  NSX-T Container Plugin natively supports Containers in VM deployment model with superior security/isolation while meeting the data plane performance requirements. The NCP can also serve as a metaplugin/CNI multiplexer that is capable of attaching multiple interfaces to Kubernetes Pods to provide multiple NIC container support.
```
Comment (CsatariGergely) [inline doc/ref_arch/kubernetes/chapters/chapter04.md]: - The link you provided gives a 404
- What does "CNCF Compliant" mean in this context? Isn't CNI compliant what you wanted to write?
- Metaplugin means, that the CNI plugin can "control" other CNI plugins. Is NCP capable of this?
- Can you please provide a link to NSX Container Plugin?

**Comment by CsatariGergely** [comment]
Sorry, but this was merged a bit [too quick](https://github.com/cntt-n/CNTT/blob/master/CONTRIBUTING.md#approvals). I think my comments were not addressed. I still believe, that there is not sutch a thing as "CNCF compliant". You did not provide a link so I could not check if the CNI plugin is open source. Now with the correct link I could doublecheck, that it is **not an open source project**. Please correct me if I'm wrong. If it is not open source, than it should not be in the RA. Also I still have a feeling that that is not a metaplugin, but a CNI frontend for an SDN.
@ramkri123 , @rabi-abdel , @tomkivlin any suggestions how to address this?

**Comment by rabi-abdel** [comment]
I don't know how to configure GitHub to overcome this. Currently, the merge button appears as long as three approvals received (including one code owner) so it is easy for a mistake to happen and a PR to be merged earlier that it should. The process we are following so far is that only code owners are anticipated to merge a PR (to make sure consensus are reached and all input is addressed).

**Comment by yangxu-hw** [comment]
I guess we need a seperate issue to fix the text if necessary.

**Comment by tomkivlin** [comment]
> I guess we need a seperate issue to fix the text if necessary.

I agree. @CsatariGergely can you create an Issue to address the questions you had please?

**Pull request merged**